### PR TITLE
Brook inventory

### PR DESCRIPTION
**Pull request by fjros**
##### ISSUE TYPE
- Feature Pull Request
##### ANSIBLE VERSION

```
ansible 2.0.0.2
  config file = 
  configured module search path = Default w/o overrides
```
##### SUMMARY

This PR adds a new dynamic inventory for the Brook.io cloud broker. It is similar to other contributed inventories. The only change is the addition of two files: `contrib/inventory/brook.py` and `contrib/inventory/brook.ini`.

The PR does not change Ansible's output in any way.

### CODE REVIEW

File: contrib/inventory/brook.py [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+
+"""
+Brook.io external inventory script
+==================================
+
+Generates inventory that Ansible can understand by making API requests to Brook.io via the libbrook
+library. Hence, such dependency must be installed in the system to run this script.
+
+The default configuration file is named 'brook.ini' and is located alongside this script. You can
+choose any other file by setting the BROOK_INI_PATH environment variable.
```
Comment (jpic) [inline contrib/inventory/brook.py:27]: Same [food for thought](https://github.com/ansible/ansible/pull/15284#discussion_r58552173) applies here.

Comment (fjros) [inline contrib/inventory/brook.py:27]: Don't have a strong preference, but `BROOK_INI_PATH` seems to be consistent with other dynamic inventories like apache-libcloud, ec2, and gce.

**Comment by fjros** [comment]
Any update on this PR??

File: contrib/inventory/brook.py [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+try:
+    import json
+except ImportError:
+    import simplejson as json
+
+try:
+    import libbrook
+except:
+    print('Brook.io inventory script requires libbrook. See https://github.com/doalitic/libbrook')
+    sys.exit(1)
```
Comment (bcoca) [inline contrib/inventory/brook.py:94]: errors should go to stderr, you can use `sys.exit("msg")` to do both in one statement

**Pull request merged**